cmd/version.go

```diff
@@ -18,3 +18,41 @@ func ShowVersion(programName string) {
 
 	fmt.Printf("go: %s\n", runtime.Version())
 }
+
+type (
+	Foo struct {
+		A int
+		B string
+	}
+	FooHasPointer struct {
+		A *int
+		B string
+	}
+)
+
+func escapeValue() *int {
+	var a int // moved to heap: a
```

Comment:
Why doesn't the escape check result appear here?

https://github.com/confidential-containers/cloud-api-adaptor/actions/runs/4163757480/jobs/7204543608#step:6:93